latest/src/app/documentation/demos/quick-list/quick-list.demo.html

```diff
@@ -0,0 +1,149 @@
+          <td class="hidden-xs-down">default</td>
+          <td class="left">The object that should refer to the no-option selected</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>[clrAllValues]</b>
+            <div class="hidden-sm-up">Type: List of ClrQuickListValue</div>
+            <div class="hidden-sm-up">Default: false</div>
+          </td>
+          <td class="left hidden-xs-down">List of ClrQuickListValue</td>
+          <td class="hidden-xs-down">empty</td>
+          <td class="left">Contains all the options list</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>[clrValues]</b>
+            <div class="hidden-sm-up">Type: List of ClrQuickListValue</div>
+            <div class="hidden-sm-up">Default: undefined</div>
+          </td>
+          <td class="left hidden-xs-down">List of ClrQuickListValue</td>
+          <td class="hidden-xs-down">empty List</td>
+          <td class="left">The value of the preselected option</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>[clrAddLabel]</b>
+            <div class="hidden-sm-up">Type: string</div>
+            <div class="hidden-sm-up">Default: ADD OPTION</div>
+          </td>
+          <td class="left hidden-xs-down">string</td>
+          <td class="hidden-xs-down">ADD OPTION</td>
+          <td class="left">Label for the add button</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>[clrMandatory]</b>
+            <div class="hidden-sm-up">Type: boolean</div>
+            <div class="hidden-sm-up">Default: false</div>
+          </td>
+          <td class="left hidden-xs-down">boolean</td>
+          <td class="hidden-xs-down">false</td>
+          <td class="left">Decides whether at least one option should be selected</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>(clrValuesChanged)</b>
+            <div class="hidden-sm-up">Type: String</div>
+            <div class="hidden-sm-up">Default: undefined</div>
+          </td>
+          <td class="left hidden-xs-down">List of ClrQuickListValue</td>
+          <td class="hidden-xs-down">undefined</td>
+          <td class="left">Event which returns the user's selected options whenever the list changes</td>
+        </tr>
+        <tr>
+          <td class="left">
+            <b>(clrEmptyOptionAdded)</b>
+            <div class="hidden-sm-up">Type: void</div>
+            <div class="hidden-sm-up">Default: undefined</div>
+          </td>
+          <td class="left hidden-xs-down">void</td>
+          <td class="hidden-xs-down">undefined</td>
+          <td class="left">Event which triggers on the add button clicked</td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+
+    <div id="code-examples">
+      <h3 id="examples">Code &amp; Examples</h3>
+      <clr-quick-list class="clr-col-4" [clrAllValues]="possibleActivityResults" [(clrValues)]="selectedActivityResults"
```

Comment:
please dont use business related variables like "activity results"

Reply:
shall I change the values as well ? 
I used some of the activity results in the tasks app

Reply:
yes please, just some generic text